SCClassLibrary/Common/Streams/Stream.sc

```diff
@@ -346,15 +346,21 @@ PauseStream : Stream {
 		isWaiting = true;	// make sure that accidental play/stop/play sequences
 						// don't cause memory leaks
 		era = CmdPeriod.era;
-		clock.play({
-			if(isWaiting and: { nextBeat.isNil }) {
-				clock.sched(0, this);
-				isWaiting = false;
-				this.changed(\playing)
-			};
-			nil
-		}, quant.asQuant);
-		this.changed(\userPlayed);
+		protect {
```

Comment:
Hmm... I was all set to say, OK, we're catching errors in dependants, but in the end, I have to stick with my original assumption that `protect` here is unnecessary.

The dependants are being notified *after* the PauseStream has updated all of its state variables. So (as far as I can see), an error in a dependant will not leave the PauseStream in an inconsistent state.

For example:

```
(
t = Task {
	inf.do { |i| i.postln; 1.wait }
};

c = SimpleController(t)
.put(\playing, { c.remove; 1.flurghle });

t.play;
)

^^ ERROR: Message 'flurghle' not understood.
RECEIVER: 1


0.0
1.0
2.0
3.0...
```

The `playing` dependant is scheduled on the clock, not under control of `protect`, so the error is not caught, `streamError` is not called, and... the Task continues on its merry way.

That is, an error in user code encountered when starting an idle PauseStream has no ill effect on the player -- so it's strictly the user's responsibility.

However, adding `protect` here introduces an inconsistency:

```
(
t = Task {
	inf.do { |i| i.postln; 1.wait }
};

c = SimpleController(t)
.put(\userPlayed, { c.remove; 1.flurghle });

t.play;
)

^^ ERROR: Message 'flurghle' not understood.
RECEIVER: 1

... waiting... hm... nothing is happening...

t.isPlaying  // false, why?
```

The fact that there is different error handling behavior for `\playing` and `\userPlayed` dependants doesn't seem intended.

Simplest solution is just to drop this one. It isn't doing any good, while creating new confusion.

Also `EventStreamPlayer` overrides PauseStream's `play` method -- that would be irrelevant after removing this `protect`, but as given, the PR wasn't quite complete.